.github/workflows/tests.yaml

```diff
@@ -1,23 +1,40 @@
 ---
 name: tests
 
-on:
-  push:
-    branches-ignore:
-    - 'development/**'
+on: push
+
+permissions:
```

Comment:
giving write permissions to the whole workflow is somewhat of a security risk, best to limit the scope of these...

can we set the permissionson a single step, or at least a single job in the workflow (essetially the same here, but not the case in other workflows/repos)?

Reply:
We can't set it at a step level, job or workflow level only. I can split the jobs in this case.

Let me know.

Reply:
job level is better already, and no point splitting this specific job. But for other repos it gives a hint/example to try and reasonably minimize what is done in the "publishing" job.

Reply:
done